### PR TITLE
GEOMESA-1716 HBase/Cassandra Doc Updates

### PR DESCRIPTION
**Pull request by jahhulbert-ccri**
Signed-off-by: Andrew Hulbert <ahulbert@ccri.com>

### CODE REVIEW

File: docs/user/cassandra/heatmaps.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Cassandra Heatmaps
+==================
+
+Cassandra can generate client-side heatmaps in GeoServer by defining a style and using the built in ``gs:Heatmap``
+process within GeoServer.
+
+To start, add a new SLD style to GeoServer named "heatmap" and use this sld
+:download:`clientside-heatmap.sld <_static/sld/clientside-heatmap.sld>`
```
Comment (mdzimmerman) [inline docs/user/cassandra/heatmaps.rst]: This link is relative, so this need to be changed to either ``<../_static/sld/clientsite-heatmap.sld>`` or ``</user/_static/sld/clientside-heatmap.sld>``.

File: docs/user/hbase/install.rst [hunk 2]
```diff
@@ -87,19 +87,11 @@ Run ``geomesa-hbase`` without arguments to confirm that the tools work.
...
-    $ mvn clean install -Phbase
-
-After building, extract ``target/geomesa-hbase-gs-plugin_2.11-$VERSION-install.tar.gz`` into GeoServer's
+The HBase GeoServer plugin is bundled by default in a GeoMesa binary distribution. To install, extract
+``$GEOMESA_HBASE_HOME/dist/gs-plugins/geomesa-hbase-gs-plugin_2.11-$VERSION-install.tar.gz`` into GeoServer's
 ``WEB-INF/lib`` directory. Note that this plugin contains a shaded JAR with HBase 1.2.3
-bundled. If you require a different version, modify the ``pom.xml`` and rebuild following
-the instructions above.
+bundled. If you require a different version, modify the ``pom.xml`` and build the geomesa hbase plugin project from
+scratch with maven.
```
Comment (mdzimmerman) [inline docs/user/hbase/install.rst]: [super picky] geomesa, hbase, maven -> GeoMesa, HBase, Maven

File: docs/user/hbase/heatmaps.rst [hunk 3]
```diff
@@ -4,7 +4,7 @@ HBase Heatmaps
 HBase can generate client-side heatmaps in GeoServer by defining a style and using the built in ``gs:Heatmap`` process
 within GeoServer.
 
-To start, add a new SLD style to GeoServer named "hbase-heatmap" and use this sld
-:download:`hbase-heatmap.sld <_static/sld/hbase-heatmap.sld>`
+To start, add a new SLD style to GeoServer named "heatmap" and use this sld
+:download:`heatmap.sld <_static/sld/clientside-heatmap.sld>`
 
-Now you can change style parameter to be styles=hbase-heatmap in the Layer Preview or PNG preview in GeoServer.
+Now you can change style parameter to be styles=heatmap in the Layer Preview or PNG preview in GeoServer.
```
Comment (mdzimmerman) [inline docs/user/hbase/heatmaps.rst]: "change *the* style parameter"

File: docs/user/cassandra/heatmaps.rst (hunk 1 above)
Comment (mdzimmerman) [inline docs/user/cassandra/heatmaps.rst]: I think this is broken in the ``user/hbase/heatmaps.rst`` file too.

File: docs/user/hbase/index.rst [hunk 4]
```diff
@@ -8,7 +8,8 @@ source distribution.
 
 .. _Apache HBase: https://hbase.apache.org/
 
-To get started with the HBase Data Store, try the :doc:`/tutorials/geomesa-quickstart-hbase` tutorial.
+To get started with the HBase Data Store, try the :doc:`/tutorials/geomesa-quickstart-hbase` tutorial or the
+:doc:`/tutorials/geomesa-hbase-s3-on-aws` tutorial for using HBase backed by S3 instead of HDFS.
```
Comment (mdzimmerman) [inline docs/user/hbase/index.rst]: This is not a problem with your PR, but ``geomesa-hbase-s3-on-aws`` isn't listed in ``/tutorials/index.rst``.

File: docs/user/hbase/install.rst [hunk 5]
```diff
@@ -111,11 +103,13 @@ GeoServer's ``WEB-INF/lib`` directory:
  * hadoop-mapreduce-client-core-2.7.3.jar
  * hadoop-yarn-api-2.7.3.jar
  * hadoop-yarn-common-2.7.3.jar
- * zookeeper-3.4.6.jar
+ * zookeeper-3.4.9.jar
  * commons-configuration-1.6.jar
 
 (Note the versions may vary depending on your installation.)
 
+You can use the bundled ``$GEOMESA_HBASE_HOME/bin/install-hadoop.sh`` script to install these jars.
```
Comment (mdzimmerman) [inline docs/user/hbase/install.rst]: [super picky] jars -> JARs

File: docs/user/hbase/index.rst (hunk 4 above)
Comment (jahhulbert-ccri) [inline docs/user/hbase/index.rst]: added it

**Comment by jahhulbert-ccri** [comment]
@mdzimmerman updated.

**Comment by mdzimmerman** [comment]
Looks good to me! When this gets merged, I will cherry-pick and add it to the 1.3.1 docs on the website.

**Pull request merged**